layouts/section/partnerships.html

```diff
@@ -68,59 +68,21 @@ <h2>{{ i18n "partnerships-team" | safeHTML }}</h2>
     <div class='row'>
       <div class='col-xs-12'>
         <h2>{{ i18n "what-were-working-on" }}</h2>
-        <p>{{ i18n "partnerships-partner-overview" }}
-
-          {{ $blog_url := "picking-our-projects" }}
-          {{ if eq $.Site.Language.Lang "fr" }}
-          {{ $blog_url = "choisir-nos-projets" }}
-          {{ end }}
-          <a href="{{with $.Site.GetPage $blog_url}}{{.Permalink}}{{ end }}">
-            {{ i18n "blog-post" }}.
-          </a>
-        </p>
       </div>
     </div>
   </div>
 
-  <!-- In-flight partnerships -->
   {{ $products := where .Site.Pages ".File.Dir" "==" "products/products/" }}
   <div class='container'>
-    <div class='row'>
-      <div class='col-xs-12'>
-        <h3>{{ i18n "partnerships-inflight" }}</h3>
-      </div>
+    <div class='row equal'>
+      {{ range $elem_index, $elem_val := $products }}
```

Comment:
Do we want to limit the number of things shown here in any way? It's currently at 14 cards I think?

Reply:
The redesign has all the cards being displayed at once. So we're no longer splitting them up.